server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/ec.js

```diff
@@ -31,6 +31,10 @@ $(document).ready(function () {
   if (sessionStorage.ecDetailsJSON === undefined) {
     sessionStorage.ecDetailsJSON = JSON.stringify([]);
   }
+
+  // display datatables errors in the console instead of in alerts
+  $.fn.dataTable.ext.errMode = 'throw';
```

Comment:
This looks like a global parameter. Do we want to set this globally? Is this okay for other tables that might be displayed on the page (if there are any)?

Reply:
The other tables will create similar alerts when the scenario outlined in #3057 happens, so I think its best that it affects all of the tables in this case.